### PR TITLE
i18n(zh-cn): page guide images

### PR DESCRIPTION
**Pull request by KyomuAoi**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

This PR adds translation of Guides/Image in Simplified Chinese.

### CODE REVIEW

**Comment by delucis** [comment]
Thanks so much for this PR @KyomuAoi! I’ve asked our other translators to take a look so we can get you some feedback.

If you use Discord, we would be happy to have you join us in our #docs-i18n channel where we have dedicated threads for all languages and translators can discuss: https://astro.build/chat

File: src/pages/zh-cn/guides/images.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: 图片
+description: 学习如何在 Astro 项目中包含图片。
+i18nReady: true
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+在 Astro 项目中，你有许多方法把图片放上自己的网站。无论是用项目中的图片文件、是链接到别处的图片，还是从像 CMS（内容管理系统）或 CDN（内容分发网络）这样的地方加载，都没问题！
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:9]:
```suggestion
在 Astro 项目中，有很多种在网站中使用图片的方式。无论是用存储在项目中的图片，还是链接外部图片，亦或从像 CMS（内容管理系统）或 CDN（内容分发网络）这样的地方加载图片，都没问题！
```

File: src/pages/zh-cn/guides/images.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+在你的 `.md` 文件中添加标准 Markdown 语法的 `![]()`，或者标准 HTML 语法的 `<img>` 标签，来调用位于 `public/` 目录下或其它服务器上的图片。以下代码样例针对两种图片分别展示了 Markdown 写法和 HTML 写法。
+
+
+```md
+// src/pages/post-1.md
+
+# 试着写个 Markdown 吧！
+
+<!-- 存放在项目中 public/assets/stars.png 路径的图片 -->
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:40]:
```suggestion
<!-- 存储在 public/assets/stars.png 的本地图片 -->
```

File: src/pages/zh-cn/guides/images.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+## 从 CMS 或 CDN 加载图片
+
+Astro 与 CMS 或 CDN 等协作流畅。通过 `<img>` 的话，只需将 URL 用作图片的 `src` 属性即可，就像你写 HTML 或者 JSX 时一样。若是使用 Astro 的「图片」集成，即 `<Image />` 组件与 `<Picture />` 组件，就按远端图片的方式，将 URL 作为 `src` 属性填入。
+
+或是你的 CDN 提供了 Node.js SDK，那么你便可以将之用于项目中。例如，[Cloudinary 的 SDK](https://cloudinary.com/documentation/node_integration) 会自动为你生成带有合适的 `src` 属性的 `<img>` 标签。
+
+## 社区开发的集成
+
+除了官方的 [`@astrojs/image`](/zh-cn/guides/integrations-guide/image/) 集成，还有着一些[社区开发的用于图片的集成](https://astro.build/integrations/css+ui/?q=image)，也能用于在你的 Astro 项目中处理和优化图片。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:229]:
```suggestion
除了官方的 [`@astrojs/image`](/zh-cn/guides/integrations-guide/image/) 集成外，社区还开发了[一些用于图片的集成](https://astro.build/integrations/css+ui/?q=image)，它们能够帮你处理并优化 Astro 项目中的图片。
```

File: src/pages/zh-cn/guides/images.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+<Image src={import('../assets/logo.png')} alt="Astro 的 logo"/>
+<Image src={rocket} width={300} alt="漂上月球，宇宙飞船。">
+<Picture src={rocket} widths=[{200, 400, 800}] sizes="(max-width: 800px) 100vw, 800px" alt="火箭发射中……" />
+<Picture src={galaxy} widths=[{200, 400, 800}] aspectRatio={16/9} sizes="(max-width: 800px) 100vw, 800px" alt="外空。" />
+```
+
+## 从 CMS 或 CDN 加载图片
+
+Astro 与 CMS 或 CDN 等协作流畅。通过 `<img>` 的话，只需将 URL 用作图片的 `src` 属性即可，就像你写 HTML 或者 JSX 时一样。若是使用 Astro 的「图片」集成，即 `<Image />` 组件与 `<Picture />` 组件，就按远端图片的方式，将 URL 作为 `src` 属性填入。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:223]:
```suggestion
Astro 与图片 CDN 可以无缝集成。就像写 HTML 或者 JSX 一样，只需将图片链接用作  `<img>` 的 `src` 属性即可。若使用 `<Image />` 组件与 `<Picture />` 组件，则是将原创链接填入 `src` 属性。
```

File: src/pages/zh-cn/guides/images.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+也可以看看 MDN 的指南：[响应式图片与美术设计](https://developer.mozilla.org/zh-CN/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction)。
+:::
+
+`<Picture />` 组件默认会在图片的原始格式之外，再包括 `avif` 和 `webp` 两种格式。
+
+#### 存放在本地的图片
+
+存放在 `src` 文件夹中的本地图片可以在主页中导入并用在 `<Picture />` 组件。`<Picture />` 组件的 `src`，`widths` 和 `sizes` 属性都必须设定。
+
+#### 位于远端的图片
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:181]:
```suggestion
#### 远程图片
```

File: src/pages/zh-cn/guides/images.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+:::tip
+也可以看看 MDN 的指南：[响应式图片与美术设计](https://developer.mozilla.org/zh-CN/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction)。
+:::
+
+`<Picture />` 组件默认会在图片的原始格式之外，再包括 `avif` 和 `webp` 两种格式。
+
+#### 存放在本地的图片
+
+存放在 `src` 文件夹中的本地图片可以在主页中导入并用在 `<Picture />` 组件。`<Picture />` 组件的 `src`，`widths` 和 `sizes` 属性都必须设定。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:179]:
```suggestion
可以在 frontmatter 中导入存储在 `src` 文件夹中的本地图片，并将其用于 `<Picture />` 组件。但 `<Picture />` 组件的 `src`，`widths` 和 `sizes` 属性不能为空。
```

File: src/pages/zh-cn/guides/images.md [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+
+Astro 的 [`<Picture />` 组件](/zh-cn/guides/integrations-guide/image/#picture-)能够给你的网站提供响应式的图片，这意味着图片可以有多种尺寸、格式和排版。你可以允许用户的浏览器基于屏幕尺寸与带宽等因素自行选择合适的尺寸、分辨率与格式，也可以定下基于 media queries 的规则约束它们。
+
+这个组件能用于呈现形式的设计，优化用户在不同尺寸屏幕上观看的体验。
+
+:::tip
+也可以看看 MDN 的指南：[响应式图片与美术设计](https://developer.mozilla.org/zh-CN/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction)。
+:::
+
+`<Picture />` 组件默认会在图片的原始格式之外，再包括 `avif` 和 `webp` 两种格式。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:175]:
```suggestion
除图片原始格式外，`<Picture />` 组件还会生成 `avif` 和 `webp` 两种格式的图片。
```

File: src/pages/zh-cn/guides/images.md [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+
+// 此时图片将被缩放到指定的宽度与高度
+<Image src={localImage} width={300} height={600} />
+<Image src={imageUrl} width={544} height={184} />
+
+// 此时图片将被缩放到指定的长宽比并转换为 avif 格式
+<Image src={localImage} aspectRatio="16:9" format="avif" />
+<Image src={imageUrl} height={200} aspectRatio="16:9" format="avif" />
+
+// 若为本地图片则可以 inline 导入
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:161]:
```suggestion
// 若为本地图片则可以在行内导入
```

File: src/pages/zh-cn/guides/images.md [hunk 9]
```diff
@@ -0,0 +1,229 @@
...
+import { Image } from '@astrojs/image/components';
+import localImage from '../assets/local.png';
+const imageUrl = 'https://astro.build/assets/logo.png';
+---
+
+// 经过优化的本地图片，但保留了原始的宽度、长度与格式
+<Image src={localImage} />
+
+// 此时图片的高度将基于原始图片的长宽比或自行指定的长宽比计算。
+// 请注意，仅当加载本地图片时，「基于原始图片的长宽比」计算方法才适用。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:149]:
```suggestion
// 请注意，仅当加载本地图片时，”基于原始图片的长宽比“计算方法才适用。
```

File: src/pages/zh-cn/guides/images.md [hunk 10]
```diff
@@ -0,0 +1,229 @@
...
+// 存放在项目中 public/assets/stars.png 路径的图片
+![一片繁星闪烁的夜空。](/assets/stars.png)
+<img src="/assets/stars.png" alt="一片繁星闪烁的夜空。" />
+
+// 位于其它服务器上的图片
+![Astro 的 logo](https://astro.build/assets/logo.png)
+<img src="https://astro.build/assets/logo.png" width="25" alt="Astro 的 logo" />
+```
+
+### 作为 UI 框架中的组件
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:72]:
```suggestion
### UI 框架中的组件
```

File: src/pages/zh-cn/guides/images.md [hunk 11]
```diff
@@ -0,0 +1,229 @@
...
+
+以符合需要的恰当语法，将图片添加为 [UI 框架中的组件](/zh-cn/core-concepts/framework-components/) （例如在 React 或 Svelte 中）。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:74]:
```suggestion
需要与之对应的语法将图片转化为 [UI 框架中的组件](/zh-cn/core-concepts/framework-components/) （例如在 React 或 Svelte 中）。
```

File: src/pages/zh-cn/guides/images.md [hunk 12]
```diff
@@ -0,0 +1,229 @@
...
+
+// 存放在项目中 src/images/rocket.svg 路径的图片
+<img src={rocket} alt="外空中的一架火箭。"/>
+
+// 存放在项目中 public/assets/stars.png 路径的图片
+![一片繁星闪烁的夜空。](/assets/stars.png)
+<img src="/assets/stars.png" alt="一片繁星闪烁的夜空。" />
+
+// 位于其它服务器上的图片
+![Astro 的 logo](https://astro.build/assets/logo.png)
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:68]:
```suggestion
![Astro 的图标](https://astro.build/assets/logo.png)
```

File: src/pages/zh-cn/guides/images.md [hunk 13]
```diff
@@ -0,0 +1,229 @@
...
+```
+
+### 作为 UI 框架中的组件
+
+以符合需要的恰当语法，将图片添加为 [UI 框架中的组件](/zh-cn/core-concepts/framework-components/) （例如在 React 或 Svelte 中）。
+
+## 存放图片的位置
+
+### `src/`
+在 `src/` 下存放的图片需要先以**相对路径**或[别名](/zh-cn/guides/aliases/)导入，然后通过图片的 `src` 属性来引用。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:79]:
```suggestion

在 `src/` 下存放的图片需要先以**相对路径**或[别名](/zh-cn/guides/aliases/)导入，然后通过图片的 `src` 属性来引用。 
```

File: src/pages/zh-cn/guides/images.md [hunk 14]
```diff
@@ -0,0 +1,229 @@
...
+
+
+```astro
+---
+// src/pages/index.astro
+
+// Access images in `src/images/`
+import logo from '../images/logo.png';
+---
+<img src={logo} width="40" alt="Astro 的 logo" />
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:89]:
```suggestion
<img src={logo} width="40" alt="Astro 的图标" />
```

File: src/pages/zh-cn/guides/images.md [hunk 15]
```diff
@@ -0,0 +1,229 @@
...
+```
+
+### `public/`
+
+Astro 项目树中的 [`public/` 目录](/zh-cn/core-concepts/project-structure/#public) 为无需在 Astro 构建过程中处理的文件与资源设置，存放于其中的图片将会被原样拷贝到构建出的文件夹中。请注意，这些图片不以导入到你的 `.astro` 文件中的方式使用（即，应直接以相对路径方式使用），且它们的 `src` 属性应填写为**对于 public/ 目录而言的相对路径**。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:94]:
```suggestion
Astro 项目树中的 [`public/` 目录](/zh-cn/core-concepts/project-structure/#public) 为无需在 Astro 构建过程中处理的文件与资源设置，存放于其中的图片将会被原样拷贝到构建出的文件夹中。请注意，这些图片不以导入到你的 `.astro` 文件中的方式使用（即，应直接以相对路径方式使用），且它们的 `src` 属性应填写为**基于 public/ 目录的相对路径**。
```

File: src/pages/zh-cn/guides/images.md [hunk 16]
```diff
@@ -0,0 +1,229 @@
...
+---
+<img src="/images/logo.png" />
+```
+
+## Astro 的「图片」集成（Astro's Image Integration）
+
+:::caution
+一旦你安装了 `@astrojs/image` 集成，你的 `.astro` 文件就不再能用标准 HTML 的 `<img>` 标签来引用存放在你的 `src` 下的图片了。所有存放在那里的图片都应该用此 Astro 集成的组件来调用。
+
+[「图片」集成指南](/zh-cn/guides/integrations-guide/image/)中有更多关于这个崭新的实验性功能的内容。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:110]:
```suggestion
[图片集成指南](/zh-cn/guides/integrations-guide/image/)中有更多关于这个崭新的实验性功能的内容。
```

File: src/pages/zh-cn/guides/images.md [hunk 17]
```diff
@@ -0,0 +1,229 @@
...
+```astro
+---
+// src/pages/index.astro
+
+// 存取放在 `public/images/` 里的图片
+---
+<img src="/images/logo.png" />
+```
+
+## Astro 的「图片」集成（Astro's Image Integration）
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:105]:
```suggestion
## Astro 的图片集成
```

File: src/pages/zh-cn/guides/images.md [hunk 18]
```diff
@@ -0,0 +1,229 @@
...
+
+在[安装这一 Astro 集成](/zh-cn/guides/integrations-guide/image/#installation)之后，你便可以导入这两个组件，并在任何你想用的地方用上它们，包括用在 `.mdx` 文件里！
+
+:::note
+`<Image />` 和 `<Picture />` 这两个组件不能用于你的 `public/` 目录下的图片。若有需要，应使用前述标准 HTML 或 Markdown 的方式来替代。
+:::
+
+### `<Image />`
+
+Astro 的 [`<Image />` 组件](/zh-cn/guides/integrations-guide/image/#image-) 使你可以优化图片，并指定宽度、长度和/或长宽比。它甚至能协助你将图片转为某一特定格式，使你无需专门处理不同类型的图片。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:123]:
```suggestion
Astro 的 [`<Image />` 组件](/zh-cn/guides/integrations-guide/image/#image-) 可以帮助帮你优化图片，并指定宽度、长度和/或长宽比。它甚至能协助你将图片转为某一特定格式，使你无需专门处理不同类型的图片。
```

File: src/pages/zh-cn/guides/images.md [hunk 19]
```diff
@@ -0,0 +1,229 @@
...
+
+当你需要保持不同终端上图片显示大小的一致，或者精细地控制图片质量（例如 logo）时，这个组件会很有用。
+
+#### 存放在本地的图片
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:127]:
```suggestion
#### 本地图片
```

File: src/pages/zh-cn/guides/images.md [hunk 20]
```diff
@@ -0,0 +1,229 @@
...
+
+存放在项目的源文件夹中的图片可以在主页中导入并用在 `<Image />` 组件的 `src` 属性。其它属性的是否填写是可选的，若是未提供，则默认为图片本身的属性。
+
+#### 位于远端的图片
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:131]:
```suggestion
#### 远程图片
```

File: src/pages/zh-cn/guides/images.md [hunk 21]
```diff
@@ -0,0 +1,229 @@
...
+
+使用位于远端的图片时，`<Image />` 的 `src` 属性需要一个完整的 URL。除此之外，你还需要：同时提供 `width` 和 `height` 属性，或同时提供以上二者之一和 `aspectRatio` 属性。
```
Comment (kecrily) [inline src/pages/zh-cn/guides/images.md:133]:
```suggestion
使用位于远端的图片时，`<Image />` 的 `src` 属性需要一个完整的链接。除此之外，你还需要：同时提供 `width` 和 `height` 属性，或同时提供以上二者之一和 `aspectRatio` 属性。
```

**Review by kecrily — CHANGES_REQUESTED** [review]

**Comment by kecrily** [comment]
There are a few points that may need attention:

- The translation of frontmatter to 主页 is a complete mistranslation. It is a proprietary term and there is no common translation for it in Chinese
- Words such as URL(链接) and logo(图标) should be translated to noon if possible when there is a Chinese equivalent for them.
- Instead of using square quotes(“”), curved quotes(「」) are generally used in Simplified Chinese. (Although many people use square quotes for typographic beauty, including myself, this is a translation, so let's use the standard curved quotation marks)
- Respect the original text as much as possible as well as short translations (e.g. where quotation marks are not used in the original text, do not add new quotation marks if you do not really have to use them)

**Comment by KyomuAoi** [comment]
Thanks for your respectable work @kecrily ! It's really a useful lesson : )
I'd like to immediately merge most of your changes, but the translation of "Astro's Image Integration" still makes me torn since I first considered it.
While the text "Astro 的图片集成" is really good—plain and simple, in my opinion it might cause a misunderstanding that the text means "The Integration (which is a process or action, rather than something like a plugin or component) of Images in Astro".
I comes from backend field and am only familiar with "module", "package" or so, so the confusion about "integration"/"集成" might not be a common situation among Astro's users? I'm not sure.
Please let me know your idea.(And sorry for my poor English.)

**Comment by delucis** [comment]
Thank you so much for your care @KyomuAoi! ❤️ 

I’ll let @kecrily reply, but if you use Discord, you’re very welcome to also join us at https://astro.build/chat where we have a dedicated `#i18n-gang-zh` thread to discuss as well (and you can chat in Chinese if it’s easier too!)

**Comment by kecrily** [comment]
Considering that there is context, and its frequent occurrence in the documentation. I don't think integration is hard to understand.

Also, considering that 集成 acts as a noun component in the utterance rather than a verb component, it should not be easily misinterpreted as a process. (In Chinese, as in English, the verb is usually placed before the noun)

**Comment by delucis** [comment]
@KyomuAoi Just saw your comment in the commit message!

> There is a little error in https://github.com/withastro/docs/pull/1285#discussion_r946422587 that says "……可以帮助帮你优化……".
Please make this suggestion again with correct text. Thanks!
(I have little knowledge about GitHub so please tell me if there is a better way to do so.)

@kecrily Do you have a suggestion? Otherwise, @KyomuAoi you can also add your own suggestion in the comment box below the existing suggestion:

image

**Comment by kecrily** [comment]
I typed wrongly. I will modify

**Comment by kecrily** [comment]
I typed wrongly. I will modify

**Review by delucis — APPROVED** [review]
Thanks so much for the contribution @KyomuAoi! Great team work with @kecrily ❤️ 

Really appreciate the care you both put into this 🙌

**Pull request merged**